docs/output.md

```diff
@@ -187,11 +187,13 @@ Optionally, the ITS region can be extracted from each ASV sequence using ITSx, a
 
 </details>
 
-### Taxonomic classification with DADA2
```

Comment:
Line 23 needs to be changed as well
in consequence line 27 will be misleading
probably put QIIME2 taxonomic classification under that headline then as well?

Reply:
Yes, that's true. Now moved qiime2 to the same headline and fixed headers.